components/LoadUser.ts

```diff
@@ -0,0 +1,42 @@
+import { baseUrl } from "../env";
+import { Dispatch, SetStateAction } from "react";
+import { User } from "../components/types/User";
+
+export const loadUser = async (
```

Comment:
Auch hier bitte in eigenen function ordner packen

Reply:
ist erledigt